### PR TITLE
Consolidate meeting logistics documents

### PR DESCRIPTION
**Pull request by dschuff**
We currently have separate documents about attending, hosting, and day-of-event process, which also have some overlap. This PR combines most of them into inperson-meetings.md and also includes vc-meetings.md for items specific to VC.

### CODE REVIEW

**Comment by dschuff** [comment]
There are a few actual changes here, mostly to clarify and remove obsolete things. For example remote attendance is basically always expected to be a thing that we should accommodate.

File: process/inperson-meetings.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+## Attending in-person meetings
+
+For in-person meetings, we request that attendees register in advance so that 
+the host may plan capacity accordingly, and so that the meeting notes may
+record who was present. 
+
+The meeting invitation will contain logistics on location, time, address,
+parking, nearby hotels, Wi-Fi, and host contact. This invitation will be sent no
+later than 2 weeks before the meeting, through a pull request on the
```
Comment (tlively) [inline process/inperson-meetings.md:15]: Do you want to keep this strictly NFC? If not, it's probably good to mention that a 3-month lead time is better or even required.

File: process/vc-meetings.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+[meetings](https://github.com/WebAssembly/meetings/tree/main/main) repository.
+
+There are also several subgroups that maintain their own meeting
+schedule and agenda. A 
+[list](https://github.com/WebAssembly/meetings/blob/main/process/subgroups.md)
+is maintained and their minutes are kept in top-level directories in the meetings
+repository.
+
+All attendees are required to be members of the
+[community group](https://www.w3.org/community/webassembly/) prior to attending.
```
Comment (tlively) [inline process/vc-meetings.md:18]: We should add this to the in-person doc as well.

File: process/inperson-meetings.md (hunk 1 above)
Comment (dschuff) [reply process/inperson-meetings.md:15]: I think it makes sense to actually update it too. I did a bit of that already.
Actually this item in particular (advance notification for the location) is mentioned in the hosting part. I don't think it's necessary to have all the logistics available that far in advance though, just basically be committed to the venue and dates.

Comment (dschuff) [reply process/inperson-meetings.md:15]: Maybe it makes sense just to remove this "2 weeks" bit entirely. You'll want the venue and dates as far in advance as possible, and at least hotel info more than 2 weeks in advance, but 2 weeks is probably fine for the rest of it.

**Review by tlively — APPROVED** [review]

**Review by agilgur5 — COMMENTED** [review]
It looks like the README links to these pages weren't updated; submitted #1705 to cover that

**Pull request merged**